catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/common/CsexraAttributeTest.java

```diff
@@ -229,12 +229,840 @@ public void testSnowDepthMaxCategoryOther() throws NitfFormatException {
     @SuppressWarnings("unchecked")
     @Test
     public void testSnowDepthMaxCategoryNotSet() throws NitfFormatException {
-        when(tre.getFieldValue(CsexraAttribute.SNOW_DEPTH_CAT)).thenThrow(NitfFormatException.class);
+        when(tre.getFieldValue(CsexraAttribute.SNOW_DEPTH_CAT_NAME)).thenThrow(NitfFormatException.class);
 
         Serializable actual = CsexraAttribute.SNOW_DEPTH_MAX.getAccessorFunction()
                 .apply(tre);
 
         assertThat(actual, nullValue());
     }
 
+    @Test
+    public void testSensorPan() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.SENSOR_NAME)).thenReturn("PAN");
+
+        Serializable actual = CsexraAttribute.SENSOR.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, is(instanceOf(String.class)));
+        assertThat(actual, is("PAN"));
+
+    }
+
+    @Test
+    public void testSensorMS() throws NitfFormatException {
```

Comment:
Does it really matter from a unit test perspective, i.e., do we need both, or could we just keep the MS one and call it `testSensorNameIsMS()`?